README.md

```diff
@@ -4,4 +4,79 @@
 
 # LaminDB - A data framework for biology
 
-Read the [docs](https://docs.lamin.ai).
+LaminDB is an open-source data framework to enable learning at scale in computational biology.
+It lets you track data transformations, curate datasets, manage metadata, and query a built-in database for biological entities & data structures.
+
+## Setup
+
+<!-- quick-setup-lamindb.md -->
+
+Install the `lamindb` Python package:
+
+```shell
+pip install 'lamindb[jupyter,bionty]'  # support notebooks & biological ontologies
+```
+
+Create a LaminDB instance:
+
+```shell
+lamin init --storage ./quickstart-data  # or s3://my-bucket, gs://my-bucket
+```
+
+Or if you have write access to an instance, connect to it:
+
+```shell
+lamin connect account/name
+```
+
+## Quickstart
+
+<!-- py-quickstart.py -->
+
+Here's how to create an artifact while tracking source code, run environment, run logs, and inputs and outputs of a script or notebook.
+
+```python
+import lamindb as ln
+
+ln.track()  # track the run of a script or notebook
+open("sample.fastq", "w").write("@r1\nACGT\n+\nIIII\n")
```

Comment:
Looks scary. Maybe a fasta is good enough and friendlier.

Reply:
Used that! Sunny also thought fasta would be good.